lib/resources/dstu2/condition.yaml

```diff
@@ -0,0 +1,133 @@
+  cardinality: 0..1
+  type: id
+  description: The logical id of the resource to update.
+  example: '{"id": "p123412"}'
+  note: The id value must match the Condition/&lt;id> value.
+  url: http://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.id
+  action: update
+
+- name: patient
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference (Patient)
+  description: Who has the condition.
+  example: |
+             {"patient": {
+                "reference":"Patient/2744010",
+                "display":"RHEUM, TEST ONE""}}
+
+- name: encounter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference (Practitioner)
+  description: Encounter during which the condition was first asserted.
+  example: |
+             {"encounter":{
+                "reference":"Encounter/1309819"}}
+
+- name: asserter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference (Practitioner)
+  description: Person who takes responsibility for asserting the existence of the condition as part of the electronic record.
+  note: Asserter must be a `Practitioner` reference.
+  example: |
+             {"asserter":{
+                "reference":"Practitioner/2770007",
+                "display":"Song, River"}}
+
+- name: dateRecorded
+  required: 'No'
+  cardinality: 0..1
+  type: date
+  description: Estimated or actual date the condition/problem/diagnosis was first entered.
+  example: '{"dateRecorded":"2013-02-08"}'
+
+- name: code
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: Identification of the condition, problem or diagnosis.
+  example: |
+             {"code":{
+                "coding":[{
+                  "system":"http://hl7.org/fhir/sid/icd-9",
+                  "code":"345.0",
+                  "display":"Generalized nonconvulsive epilepsy",
+                  "userSelected":true}],
+                "text":"Absence seizures"}}
+
+- name: category
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: A category assigned to the condition. e.g. complaint | symptom | finding | diagnosis.
+  example: |
+             {"category":{
+                "coding":[{
+                  "system":"http://hl7.org/fhir/condition-category",
+                  "code":"diagnosis",
+                  "display":"Diagnosis"}],
+                "text":"Diagnosis"}}
+
+- name: verificationStatus
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: The verification status of the condition.
+  example: '{"verificationStatus":"confirmed"}'
+
+- name: severity
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Assessment of the severity of the condition as evaluated by the clinician.
+  example: |
+             {"severity":{
+                "coding":[{
+                  "system":"http://snomed.info/sct",
+                  "code":"371923003",
+                  "display":"Mild to moderate"}],
+                "text":"Mild to moderate"}}
+
+- name: onsetDateTime
```

Comment:
onsetDateTime, abatementDateBoolean, and abatementDateTime don't follow the standard naming convention because of the [x] pattern and will need to be linked directly using the url attribute. Currently they just link to the top of the document